generators/server/templates/src/main/java/package/service/UserService.java.ejs

```diff
@@ -1017,15 +1017,17 @@ public class UserService {
     private static <%= databaseTypeNo ? asDto('AdminUser') : asEntity('User') %> getUser(Map<String, Object> details) {
         <%= databaseTypeNo ? asDto('AdminUser') : asEntity('User') %> user = new <%= databaseTypeNo ? asDto('AdminUser') : asEntity('User') %>();
         Boolean activated = Boolean.TRUE;
+        String sub = String.valueOf(details.get("sub"));
+        String username = String.valueOf(details.get("preferred_username")).toLowerCase();
```

Comment:
In the original code, we read this value in a null check enclosed block. In case of null, `toLowerCase()` invocation will throw NPE

Reply:
Should be fixed now.